challenges/KICKin-Rollin-Challenge.tex

```diff
@@ -0,0 +1,63 @@
+% !TeX root = ../SPL-Challenges.tex
+% !TeX spellcheck = en_US
+\section{KICKin\textquotesingle \& Rollin\textquotesingle Challenge}
+
+The objective of this challenge is for a robot to successfully kick a rolling ball from a ramp into a designated goal area. 
+Each participating team will record three attempt sets, with a significant time between them.
+A more specific schedule will be communicated closer to the competition.
+An attempt sets consist of three kick attempts. 
```

Comment:
The distinction between "attempt set" and "kick attempt" is nice, but can we find a way to completely remove the word "attempt" from one of the two denominations, for maximum clarity? Maybe an "attempt set" can simply become a "session" or similar, or maybe a "kick attempt" can become just a "kick"?